app/lib/default_error_messages.rb

```diff
@@ -50,6 +50,14 @@ def self.generate!(service_type:)
         auto_wait: true,
         service_type: service_type_name
       )
+      EfileError.find_or_create_by!(
+        code: "AN UNFIXABLE ERROR",
+        message: "Something about your return disqualifies you from filing",
+        source: :intake,
+        expose: true,
+        auto_cancel: true,
+        service_type: service_type_name
+      )
```

Comment:
meant for this to be in my other PR actually but i'm pretty sure these errors are used just in the seeder and i wanted access to an auto_cancel error locally without having to manually create one every time

Reply:
[q] I don't fully understand how you used this in this PR so would love to chat about it 🙏

Reply:
discussed in person:
> Arin: how do you link this error to your local return?
> Jenny: oh so that button i added for "rejected autocancel" will find the first auto-cancel error so it end up being this one because it's the only one i have